hydra-node/test/Hydra/CryptoSpec.hs

```diff
@@ -24,7 +24,11 @@ specSigningKey :: Spec
 specSigningKey =
   describe "SigningKey" $ do
     it "show includes escaped hex" $
-      show (generateSigningKey "aaa") `shouldContain` "\"616161"
+      show (generateSigningKey "aaa") `shouldContain` "\"983487"
+    it "can be generated when seed exceeds the max seed size for algorithm" $
+      let exceedingSizeSeedA = show $ replicate 32 'x' <> "a"
+          exceedingSizeSeedB = show $ replicate 32 'x' <> "b"
```

Comment:
`show` should not be needed here. Usually it will wrap a bytestring into quotes. As this is likely the overloaded version from relude (https://hackage.haskell.org/package/relude-1.1.0.0/docs/Relude-String-Conversion.html#v:show), it will use the `IsString` of `ByteString`. That likely is `Data.ByteString.Char8.pack`. The extra `"` in the beginning and end will still result in 2 + 32 + 1 length seeds.. which is still okay, but not exactly what you intended to write here.

Should: Don't use show here, as it will include extra `"`